_source/_posts/2024-02-29-3pc.md

```diff
@@ -0,0 +1,65 @@
+
+Okta uses cookies to let applications introspect and extend user sessions. Cookies aren't required for basic login functionality. 
+
+### Web applications that rely on the Okta session for user context.  
+
+If an application hosted on your domain (`mycompanyapp.com`) redirects to your Okta subdomain (`mycompany.okta.com`) for login and then returns users to your own domain, 3rd party cookie restrictions will limit how your app can introspect or extend the Okta session. 
+
+
+### Customer-hosted Okta Sign-In Widget and customer-built login applications
+
+If you're hosting your own sign-in experience on a separate top level domain from your main app, you may be using 3rd party cookies. You might be hosting your own sign-in experience by cloning the Okta Sign-In Widget from GitHub or installing it from NPM to embed in your application, or you might have built your own custom sign-in using Okta's APIs. 
+
+If your sign-in experience is hosted on the same top-level domain as your application, 3rd party cookie deprecation won't affect its behavior. 
+
+If the sign-in experience and app are on different top-level domains, 3rd party cookie deprecation will break its ability to introspect and extend sessions, because these features use cookies. Authentication will still be possible, and tokens will still be returned, becasue these features do not rely on cookies. 
+
+### "Remember me" Features
+
+"Remember Last Used Factor" (RLUF), "Keep me signed in" (OIE) and "Remember me" (Classic) will not function when the login application is on a 3rd party domain.  
+
+## When will this affect you? 
+
+Google has made an exemption for Okta's 3rd party cookies until the end of 2024. However, you can set Chrome's flags to simulate how the browser will treat Okta's 3rd party cookies after that exemption ends. 
+
+### Test your application today!
+
+To simulate how Chrome will treat Okta's 3rd party cookies in 2025 and beyond, follow [Okta's help center guide](https://support.okta.com/help/s/article/deprecation-of-3rd-party-cookies-in-google-chrome?language=en_US). 
+
+## What's next? 
+
+Here on the Okta Developer Blog, we'll keep you up to date about how to mitigate each type of 3rd party cookie impact.
+
+* Learn more about [how blocking 3rd party cookies affects Okta environments](https://support.okta.com/help/s/article/FAQ-How-Blocking-Third-Party-Cookies-Can-Potentially-Impact-Your-Okta-Environment?language=en_US)
+* See the [session cookies guide](https://developer.okta.com/docs/guides/session-cookie/main/) for more on how cookies are used
+* [Use Chrome's feature flags](https://support.okta.com/help/s/article/deprecation-of-3rd-party-cookies-in-google-chrome?language=en_US) to test your login experience with 3rd party cookies disabled.  
```

Comment:
Add "Okta" in front of "session cookies guide."

The third item in this list has a period, whereas the items above do not. Add periods to the preceding items.